### PR TITLE
Spring expression addition

### PR DESCRIPTION
**Pull request by adkumar1**

### CODE REVIEW

**Review by dvasunin — APPROVED** [review]
just a library upgrade

File: DEPENDENCIES [hunk 1]
```diff
@@ -190,6 +190,7 @@ maven/mavencentral/org.springframework/spring-beans/6.0.7, Apache-2.0, approved,
 maven/mavencentral/org.springframework/spring-context/6.0.7, Apache-2.0, approved, #5936
 maven/mavencentral/org.springframework/spring-core/6.0.7, Apache-2.0 AND BSD-3-Clause, approved, #5948
 maven/mavencentral/org.springframework/spring-expression/6.0.7, Apache-2.0, approved, #3284
```
Comment (Siegfriedk) [inline DEPENDENCIES:192]: How can this be? Do you now really have both versions in your repo or did you not run the dash tool properly?

**Review by Siegfriedk — CHANGES_REQUESTED** [review]

File: DEPENDENCIES (hunk 1 above)
Comment (Wulghash) [reply DEPENDENCIES]: There was an error with running the tool, you are correct, we only use new version.

Comment (Siegfriedk) [reply DEPENDENCIES]: @adkumar1 @dvasunin @Wulghash make sure to never do this manually somehow! ever again!

**Pull request merged**